src/main/java/com/pnudev/springonlinedelivery/config/JwtService.java

```diff
@@ -0,0 +1,71 @@
+package com.pnudev.springonlinedelivery.config;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtService {
+    private static final String SECRET_KEY = "404D635166546A576D5A7134743777217A25432A462D4A614E645267556B5870";
```

Comment:
Move it as property and inject here as `@Value`. You mustn't hardcode sensitive values like this.